src/testing/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@o1js/testing",
+  "description": "o1js testing utilities",
+  "version": "0.0.1",
+  "type": "module",
+  "scripts": {
+    "build": "tsc -p ../../tsconfig.testing.json &&  node scripts/copy-bindings.js",
+    "prepublishOnly": "rimraf ./dist && npm run build"
+  },
+  "keywords": [
+    "o1js",
+    "testing",
+    "zk"
+  ],
+  "author": "o1labs",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/o1-labs/o1js.git",
+    "directory": "src/testing"
+  },
+  "bugs": "https://github.com/o1-labs/o1js/issues",
+  "main": "dist/node/testing/index.js",
+  "types": "dist/node/testing/index.d.ts",
+  "exports": {
+    "types": "./dist/node/testing/index.d.ts",
+    "node": "./dist/node/testing/index.js",
+    "default": "./dist/node/testing/index.js"
+  },
+  "files": [
```

Comment:
We are including mina-signer and other internal dependencies in the package when we publish it to npm. While this might not be a big size increase in the overall package (given that the compile artefacts alone are about 45mb), it's still suboptiomal to include files that have nothing to do with the actual library in the released package

Reply:
If or when we end up splitting everything into seperate `package` folders, we should definitely clean this up. So let's make sure we keep this in mind, I approve in hopes that we will follow up on this soon